Lab10/src/main/java/org/example/entities/Abiturient.java

```diff
@@ -0,0 +1,97 @@
+package org.example.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.InputMismatchException;
+import java.util.Locale;
+import java.util.Scanner;
+
+@Entity
+@Table(name = "abiturients")
+@Setter
+@Getter
+public class Abiturient {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long ID;
+    @Column(name = "lastname")
+    private String lastName;
+    @Column(name = "firstname")
+    private String firstName;
+    @Column(name = "fathername")
+    private String fatherName;
+    @Column(name = "email")
+    private String email;
+    @Column(name = "phone")
+    private String phone;
+    @Column(name = "average_point")
+    private double averagePoint;
+    @Column(name = "address")
+    private String address;
+    @Column(name = "need_hostel")
+    private boolean needHostel;
+
+    public Abiturient() {
+    }
+
+    public Abiturient(String lastName, String firstName, String fatherName, String email, String phone, double averagePoint, String address, boolean needHostel) {
+        this.lastName = lastName;
+        this.firstName = firstName;
+        this.fatherName = fatherName;
+        this.email = email;
+        this.phone = phone;
+        this.averagePoint = averagePoint;
+        this.address = address;
+        this.needHostel = needHostel;
+    }
```

Comment:
Considering that you are already using Lombok annotations it might be a good idea to use `@NoArgsConstructor` and `@RequiredArgsConstructor` annotations to avoid this boilerplate constructor code
```suggestion
```

Reply:
I agree with `@NoArgsConstrucor`.
But pay attention to parameterized constructor, it doesn't contain `ID` field.
So if I use `@RequiredArgsConstructor`, I should add `@NonNull` annotation to all fields except `ID`